cmd/cluster_deploy.go

```diff
@@ -3,11 +3,12 @@ package cmd
 import (
 	"context"
 	"fmt"
-	"github.com/pkg/errors"
```

Comment:
If you could remove this file diff, as it's only an automatic reformat (and it will spare me some time when I'll rebase my big refacto :smiling_face_with_tear: )

Reply:
done